static/js/typeahead_helper.js

```diff
@@ -95,6 +95,28 @@ function prefix_sort(query, objs, get_item) {
 
 }
 
+function split_by_subscribers(objs) {
+    var subscribers = [];
+    var non_subscribers = [];
+    var current_stream = compose.stream_name();
+
+    _.each(objs, function (person) {
+        // Home page has a stream name of ""
```

Comment:
For the Home page case, we should probably have a special case before the loop that just returns `{subs: persons, non_subs: []}`